coalib/bears/requirements/DistributionRequirement.py

```diff
@@ -0,0 +1,49 @@
+from coalib.bears.requirements.PackageRequirement import PackageRequirement
+import platform
+
+
+class DistributionRequirement(PackageRequirement):
+    """
+    This class is a subclass of ``PackageRequirement``, and helps specifying
+    distribution specific requirements, without using the manager name.
+    """
+
+    def __init__(self, **manager_commands):
+        """
+        Constructs a new ``DistributionRequirement``, using the
+        ``PackageRequirement`` constructor.
+
+        >>> dr =  DistributionRequirement(apt_get='libclang', dnf='libclangg')
+        >>> dr.package['apt_get']
+        'libclang'
+        >>> dr.package['dnf']
+        'libclangg'
+
+        :param manager_commands: comma separated (manager='package') pairs.
+        """
+        self.package = manager_commands
+
+    def install_command(self):  # pragma: no cover
```

Comment:
me neither, but do you have a better idea of testing system dependant functions? :D that would work on both circleci and all OSs that are used by everyone? :(